spec/support/miq_ae_mock_object.rb

```diff
@@ -1,11 +1,8 @@
 module Spec
   module Support
     class MiqAeMockObject
-      attr_reader :parent
       attr_reader :children
-      attr_accessor :namespace
-      attr_accessor :klass
-      attr_accessor :instance
+      attr_accessor :namespace, :klass, :instance, :parent
```

Comment:
@mkanoor This looks good to me, just looking for a second review.

Reply:
👍